### PR TITLE
add travis support

### PR DESCRIPTION
**Pull request by ExtraGravity**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+sudo: required
+
+before_install:
+    - sudo apt-add-repository -y ppa:ubuntu-toolchain-r/test
+    - sudo apt-add-repository -y ppa:beineri/opt-qt542
+    - sudo apt-get -qq update
+    - sudo apt-get -qq install g++-4.8 libc6-i386 qt54tools qt54svg qt54webkit qt54serialport
```
Comment (Parasrah) [inline .travis.yml:7]: Would it make sense to pipe the error log to a text file here

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    - sudo apt-get -qq update
+    - sudo apt-get -qq install g++-4.8 libc6-i386 qt54tools qt54svg qt54webkit qt54serialport
+    - sudo cp /usr/bin/g++-4.8 /usr/bin/g++
+    - PATH="/opt/qt54/bin:$PATH"
+    - qt54-env.sh
+    - cd ..
+    - mv ./Hermes ./src
+    - mkdir Hermes
+    - mv ./src ./Hermes/
+    - mkdir ./Hermes/build/
```
Comment (Parasrah) [inline .travis.yml]: `mkdir -p ./Hermes/build/.lib`

 would perform 15 & 16

File: .travis.yml (hunk 1 above)
Comment (ExtraGravity) [inline .travis.yml:7]: No, because we don't have access to the files after a travis build, only the logs.

File: .travis.yml (hunk 2 above)
Comment (ExtraGravity) [inline .travis.yml]: I learned about `-p` today! Good call.

File: .travis.yml (hunk 1 above)
Comment (Parasrah) [inline .travis.yml:7]: Ahh, I'll have to go read more about how a travis build works, I only read up on it briefly

**Pull request merged**